src/main/kotlin/com/chattriggers/ctjs/commands/CTCommand.kt

```diff
@@ -142,12 +143,7 @@ internal object CTCommand {
 
     private fun openFileLocation() {
         try {
-            val path = listOf(File(".").absolutePath, "config", "ChatTriggers", "modules").joinToString(File.separator)
-            if (UDesktop.isWindows) {
-                ProcessBuilder("explorer.exe", "/select,$path")
-            } else {
-                ProcessBuilder("xdg-open", path)
-            }.start()
+            UDesktop.browse(ModuleManager.modulesFolder.toURI())
```

Comment:
This is actually different than legacy, which opened the `config/ChatTriggers` folder. I'm fine with it, but we should mention it in the migration doc

Reply:
Oh I forgot that's how it used to work. I can make it back to that if you want

Reply:
Nah I think this makes sense, since the only other things in that folder are ct internal stuff, like the copied js files and I think the "master" assets folder